src/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.search__field {
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  height: 70px;
+  padding-left: 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: "Avenir", sans-serif;
+  outline: none;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+}
+
+.search__field--small {
+  height: 42px;
+  padding-left: 33px;
+  background-position: 13px 15px;
+  background-size: 11px 11px;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search__field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__field:focus {
+  background: url(./images/Search.svg) linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
```

Comment:
invalid. You don't need to change an image
![Screenshot 2023-04-03 at 21 17 17](https://user-images.githubusercontent.com/117587463/229593399-784d8915-60f8-461d-8f2c-2b85953880e1.png)